core/controllers/contributor_dashboard_admin.py

```diff
@@ -594,7 +595,17 @@ class ContributorDashboardAdminStatsHandler(
                 },
                 'default_value': None
             },
-            'max_days_since_last_activity': {
+            'max_days_since_start_date_for_last_activity': {
+                'schema': {
+                    'type': 'int',
+                    'validators': [{
+                        'id': 'is_at_least',
+                        'min_value': 0
+                    }]
+                },
+                'default_value': None
+            },
+            'max_days_since_end_date_for_last_activity': {
```

Comment:
Maybe the handler itself should take dates, instead of days? This argument name is strange, and is not very clear to me. A much better api would be if there were "start_date" and "end_date" for this handler. 

I honestly can't come up with a name that is clear + describes what you want. I'd prefer to see "calculation_start_date" and "calculation_end_date" or something like that.

Reply:
Hello @nikitaevg , made the suggested changes, PTAL